packages/backend/server/src/base/job/queue/config.ts

```diff
@@ -1,61 +1,84 @@
 import { QueueOptions, WorkerOptions } from 'bullmq';
 
-import {
-  defineRuntimeConfig,
-  defineStartupConfig,
-  ModuleConfig,
-} from '../../config';
+import { defineModuleConfig, JSONSchema } from '../../config';
 import { Queue } from './def';
 
-declare module '../../config' {
-  interface AppConfig {
-    job: ModuleConfig<
-      {
-        queue: Omit<QueueOptions, 'connection'>;
-        worker: Omit<WorkerOptions, 'connection'>;
-      },
-      {
-        queues: {
-          [key in Queue]: {
-            concurrency: number;
-          };
-        };
-      }
-    >;
+declare global {
+  interface AppConfigSchema {
+    job: {
+      queue: ConfigItem<Omit<QueueOptions, 'connection' | 'telemetry'>>;
+      worker: ConfigItem<{
+        defaultWorkerOptions: Omit<WorkerOptions, 'connection' | 'telemetry'>;
+      }>;
+      queues: {
+        [key in Queue]: ConfigItem<Omit<WorkerOptions, 'connection'>>;
+      };
+    };
   }
 }
 
-defineStartupConfig('job', {
+const schema: JSONSchema = {
+  type: 'object',
+  properties: {
+    concurrency: { type: 'integer' },
+  },
+};
+
+defineModuleConfig('job', {
   queue: {
-    prefix: AFFiNE.node.test ? 'affine_job_test' : 'affine_job',
-    defaultJobOptions: {
-      attempts: 5,
-      // should remove job after it's completed, because we will add a new job with the same job id
-      removeOnComplete: true,
-      removeOnFail: {
-        age: 24 * 3600 /* 1 day */,
-        count: 500,
+    desc: 'The config for job queues',
+    default: {
+      prefix: 'affine_job',
+      defaultJobOptions: {
+        attempts: 5,
+        // should remove job after it's completed, because we will add a new job with the same job id
+        removeOnComplete: true,
+        removeOnFail: {
+          age: 24 * 3600 /* 1 day */,
+          count: 500,
+        },
       },
     },
+    link: 'https://api.docs.bullmq.io/interfaces/v5.QueueOptions.html',
```

Comment:
Hope we can remember to make changes here when the bullmq version is upgraded later